core/src/main/java/com/google/cloud/sql/core/InstanceData.java

```diff
@@ -22,7 +22,7 @@
 import javax.net.ssl.SSLContext;
 
 /** Represents the results of a certificate and metadata refresh operation. */
-class InstanceData {
+class InstanceData implements ExpiringData {
```

Comment:
Why do we think this interface is a useful addition?

Reply:
This enables the type parameter on Refresher.